js/src/tooltip.js

```diff
@@ -284,19 +311,37 @@ const Tooltip = (() => {
 
         const container = this.config.container === false ? document.body : $(this.config.container)
 
-        $(tip).data(this.constructor.DATA_KEY, this)
+        $tip.data(this.constructor.DATA_KEY, this)
 
         if (!$.contains(this.element.ownerDocument.documentElement, this.tip)) {
-          $(tip).appendTo(container)
+          $tip.appendTo(container)
         }
 
         $(this.element).trigger(this.constructor.Event.INSERTED)
 
+        let newOffset = this.config.offset
+
+        const arrowOffset = $arrow.width() + this.config.arrowPadding
+
+        if (!newOffset) {
+          switch (OffsetMap[placement.toUpperCase()]) {
+            case +1:
+              newOffset = `+50%p - ${arrowOffset}px`
+              break
+            case -1:
+              newOffset = `-50%p + ${arrowOffset}px`
+              break
+            default:
+              newOffset = 0
+          }
+
+        }
+
```

Comment:
Can you move that part of this code in a private function please ? (offset calculation)

Reply:
Yes :-)
A small question - how do I prevent the build css- and js-files in `/dist` and `js/dist` from  being included in the PR?

Reply:
you can revert those changes thanks to : `git checkout dist/ && git checkout js/dist/` :wink:

Reply:
Just don't commit them. You can decide what files you want git to stage